### PR TITLE
AO-20351-Update-Documentation

### PR DESCRIPTION
**Pull request by ronilan**
This Pull Request updated the README document and adds initial versions of docs for Release Process and GitHub Actions.

The two commits are result of rebasing multiple online editing commits. Branch stays open for further iterations. Feel free to participate.

### CODE REVIEW

File: docs/release-process.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  - Release Notes for Agent
+  - Release Notes for Bindings (ONLY IF needed)
+  - Link to Jira Release.
+  - Link to GitHub Action Agent Prerelease Run
+  - Link to Githb Action Bindings Prerelease Run (if needed).
+  - Link to commits since last Agent release.
+  - Link to commits since last Binding release (if needed).
+  - Link to Checkmarx scan for Agent and review of any findings.
+  - Link to Checkmarx scan for Bindings and review of any findings.
+  - Link to Dependabot alrets for Agent and review of any.
```
Comment (cheempz) [inline docs/release-process.md:61]: Typo `alrets`

File: docs/release-process.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  - ```git push --tags```
+
+### Agent
+1. Create the release branch
+  - ```git checkout -b Release```
+2. Update [CHANGELOG.md](https://github.com/appoptics/appoptics-apm-node/blob/master/CHANGELOG.md)
+4. Update bindings version **ONLY IF needed**
+  - ```npm install @appoptics/apm-bindings@latest```
+  - ```git commit -am "Updated @appoptics/apm-bindings to latest."```
+5. Bump and tag prerelease version
```
Comment (cheempz) [inline docs/release-process.md:86]: Should be  *release* version right?

**Review by cheempz — COMMENTED** [review]
Spotted a few typos :)

File: docs/release-process.md (hunk 2 above)
Comment (ronilan) [reply docs/release-process.md]: 👍

File: docs/release-process.md (hunk 1 above)
Comment (ronilan) [reply docs/release-process.md]: 👍

**Pull request merged**